Apps/Examples/Examples/Info.plist

```diff
@@ -45,5 +45,7 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>CADisableMinimumFrameDurationOnPhone</key>
```

Comment:
Does this have any effect on the frame rate of examples other than DebugMap example?